packages/m2m-gateway/src/api/purposeTemplateApiConverter.ts

```diff
@@ -0,0 +1,21 @@
+import { m2mGatewayApi, purposeTemplateApi } from "pagopa-interop-api-clients";
+
+export function toM2MGatewayApiPurposeTemplate(
+  purposeTemplate: purposeTemplateApi.PurposeTemplate
+): m2mGatewayApi.PurposeTemplate {
+  return {
```

Comment:
Here it's missing `purposeRiskAnalysisForm`, is it part of a further endpoint?

Reply:
Just saw this comment, will be a dedicated route: https://github.com/pagopa/interop-be-monorepo/pull/2540#discussion_r2478912035